src/components/PuzzleModal/index.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { useState } from "react"
+import "./index.scss"
+import { ReactComponent as ExitIcon } from "../../images/icons/circle-xmark-solid.svg"
+
+interface PuzzleProps {
+  children: React.ReactNode
+  question: string
+  answer: string
+}
+
+const link =
+  "https://docs.google.com/forms/d/e/1FAIpQLSfyGGDipFozDbQrmDBIoGvCwLcn5Lec3GMNnCgsRWKp9Q4syA/viewform"
+
+const PuzzleModal: React.FC<PuzzleProps> = ({
+  children,
+  question,
+  answer,
+}: PuzzleProps) => {
+  const [response, setResponse] = useState("")
+  const [success, setSuccess] = useState(false)
+  const [errorMessage, setErrorMessage] = useState("")
+  const [isPuzzleHidden, setIsPuzzleHidden] = useState(true)
+  const handleExit = () => setIsPuzzleHidden(true)
+  const hiddenOrNot = isPuzzleHidden
+    ? "puzzlemodal__container hidden"
+    : "puzzlemodal__container displayed"
+
+  const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setResponse(event.target.value)
+  }
+
+  const handleSubmit = (event: React.KeyboardEvent<HTMLDivElement>) => {
+    if (event.key === "Enter") {
+      if (response.toLowerCase() === answer) {
+        setSuccess(true)
+        setErrorMessage("")
+      } else {
+        setErrorMessage("Sorry, that's incorrect. Guess again")
+      }
+    }
+  }
+  const SuccessMessage = (
+    <div className='puzzlemodal__container--success'>
+      That&apos;s correct!&#8239;
+      <a href={link} target='_blank' rel='noreferrer'>
```

Comment:
Also, I wanted to ask if this should've been blue?

<img src="https://user-images.githubusercontent.com/20177171/147615893-805aa80a-8989-46fe-afb9-85c678c3c59e.png" width="200px">

Reply:
Good point, I changed it to white but past that I'm not too sure what we want stying wise. I can definitely use an icon instead too